src/test/java/com/fleet/utilities/RandomUserGenerator.java

```diff
@@ -0,0 +1,53 @@
+package com.fleet.utilities;
+
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class RandomUserGenerator {
+    public static String getUserName(String userType){
+        String username = "";
+        int minimum,maximum;
+
+        int random = new Random().nextInt(1,3);
+
```

Comment:
please change the "new Random()" to "ThreadLocalRandom.current()"  everywhere in this class because the random class is not compatible with the project's version of java